css-brochure-form/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Learn More</h1>
+    <div>
+      <label for="CompanyEmail" id="email1">Company Email</label>
+    </div>
+    <div>
+      <input type="email" id="CompanyEmail" value="sally@company.co" />
+    </div>
+    <div>
+      <label for="name">Name</label>
+    </div>
+    <div>
+      <input type="text" id="name" value="Sally Smith" />
+    </div>
+    <div>
+      <label for="hiring">Where are you hiring?</label>
+    </div>
+    <div>
+      <input type="text" id="hiring" value="City, Country" />
+    </div>
+    <div>
+      <label for="number"
+        >How many engineers are you hiring in the next 6 months?</label
+      >
+      <div>
+        <select id="number" name="engineers">
+          <option>1-2 Engineers</option>
+          <option>2-3 Engineers</option>
+          <option>3-4 Engineers</option>
+        </select>
+      </div>
+      <br />
```

Comment:
We don't want to use `<br />` for spacing purposes. We should utilize `margin` to space out elements